src/accordion.tsx

```diff
@@ -194,19 +199,22 @@ const AccordionItemContent = React.forwardRef<TouchableElement, AccordionItemCon
 );
 
 export const AccordionItem = React.forwardRef<TouchableElement, AccordionItemContentProps>(
-    ({dataAttributes, ...props}, ref) => (
-        <AccordionItemContent
-            {...props}
-            ref={ref}
-            dataAttributes={{'component-name': 'AccordionItem', ...dataAttributes}}
-        />
+    ({dataAttributes, role, ...props}, ref) => (
+        <div role={role} className={sprinkles({width: '100%'})}>
+            <AccordionItemContent
+                {...props}
+                ref={ref}
+                dataAttributes={{'component-name': 'AccordionItem', ...dataAttributes}}
+            />
+        </div>
     )
 );
 
 type AccordionBaseProps = {
     children: React.ReactNode;
     dataAttributes?: DataAttributes;
     onChange?: (index: number, value: boolean) => void;
+    role?: string;
```

Comment:
Initially, I added "list" and "listitem" as default roles for Accordion and AccordionItem, but after discussing with @aweell, he pointed out that it may not be a good idea.

For example, if the accordions are being used to divide some content of a document in different sections, it's not the best idea to use a list for them.

We've decided to make it customizable, but without setting any default values

Reply:
Specs are updated accordingly